src/combinatorics/simple_graph/basic.lean

```diff
@@ -204,6 +302,19 @@ lemma complete_graph_is_regular [decidable_eq V] :
   (complete_graph V).is_regular_of_degree (fintype.card V - 1) :=
 by { intro v, simp }
 
+/--
+The minimal degree of all vertices
+-/
+def min_deg (G : simple_graph V) [nonempty V] [decidable_rel G.adj] : ℕ :=
+finset.min' (univ.image (λ (v : V), G.degree v)) (nonempty.image univ_nonempty _)
+
+/--
+The maximal degree of all vertices
+-/
+def max_deg (G : simple_graph V) [h : nonempty V] [decidable_rel G.adj]: ℕ :=
+finset.max' (univ.image (λ (v : V), G.degree v)) (nonempty.image univ_nonempty _)
```

Comment:
I think these should be called `min_degree` and `max_degree` - and possibly the docstring should say `minimum` and `maximum` as well.